src/java/org/apache/cassandra/cql3/ColumnSpecification.java

```diff
@@ -28,17 +28,28 @@
 
 public class ColumnSpecification
 {
+    protected static final String EMPTY_COMMENT = "";
+    protected static final String EMPTY_SECURITY_LABEL = "";
     public final String ksName;
     public final String cfName;
     public final ColumnIdentifier name;
     public final AbstractType<?> type;
+    public final String comment;
+    public final String securityLabel;
 
     public ColumnSpecification(String ksName, String cfName, ColumnIdentifier name, AbstractType<?> type)
+    {
+        this(ksName, cfName, name, type, EMPTY_COMMENT, EMPTY_SECURITY_LABEL);
+    }
+
+    public ColumnSpecification(String ksName, String cfName, ColumnIdentifier name, AbstractType<?> type, String comment, String securityLabel)
     {
         this.ksName = ksName;
         this.cfName = cfName;
         this.name = name;
         this.type = type;
+        this.comment = comment == null ? EMPTY_COMMENT : comment;
+        this.securityLabel = securityLabel == null ? EMPTY_SECURITY_LABEL : securityLabel;
```

Comment:
Empty string is assigned when the value is null. Then, how do un-setting comment and security label queries work?